src/main/java/org/broadinstitute/hellbender/utils/config/CustomBooleanConverter.java

```diff
@@ -0,0 +1,16 @@
+package org.broadinstitute.hellbender.utils.config;
+
+import org.aeonbits.owner.Converter;
+
+import java.lang.reflect.Method;
+
+/**
+ * Created by jonn on 7/21/17.
+ */
+public class CustomBooleanConverter implements Converter<Boolean> {
+
+    @Override
+    public Boolean convert(Method method, String input) {
```

Comment:
This method exists to allow for the Owner parser to parse the string value with spaces at the end.  I initially made it as an example of how to do that.  It may be worth creating a class like this for all the types if we want to allow for parsing spaces appended to values that are not strings.